citadel/indico_citadel/plugin.py

```diff
@@ -75,17 +82,26 @@ class CitadelPlugin(LiveSyncPluginBase):
         'num_threads_files': 5,
         'num_threads_files_initial': 25,
         'disable_search': False,
+        'large_category_warning_threshold': 0,
     }
     backend_classes = {'citadel': LiveSyncCitadelBackend}
 
     def init(self):
         super().init()
         self.connect(signals.core.get_search_providers, self.get_search_providers)
         self.connect(signals.plugin.cli, self._extend_indico_cli)
+        self.template_hook('category-protection-page', self.check_event_categories)
 
     def get_search_providers(self, sender, **kwargs):
         from indico_citadel.search import CitadelProvider
         return CitadelProvider
 
     def _extend_indico_cli(self, sender, **kwargs):
         return cli
+
+    def check_event_categories(self, category):
```

Comment:
I usually start those methods with an underscore since they are internal to the plugin.